controllers/complianceeventsapi/migrations/000001_compliance_history_initial_tables.up.sql

```diff
@@ -0,0 +1,57 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS clusters(
+   id serial PRIMARY KEY,
+   name TEXT NOT NULL,
```

Comment:
It may be in the design and I don't remember, are we using the namespaced name so that's why there's no namespace here?

Reply:
I think you meant to post this on the `parent_policies` table. Yes, we are using the `<namespace>.<name>` format for the name.

I think it makes sense to have a nullable `namespace` column on the `policies` table though to later support integrations with namespaced policies such as Kyverno.